libdivide.h

```diff
@@ -18,9 +18,51 @@
 #include <stdint.h>
 
 #if defined(__cplusplus)
+#if __AVR__
```

Comment:
Please make this `#if defined(__AVR__)` matching the check below.

Reply:
Redundant with latest changes